content/07.pathogenesis.md

```diff
@@ -10,6 +10,36 @@ This section would also be great for the introduction of zoonotic diseases which
 [What cells are primary infection sites for SARS-CoV-2?]
 [What structural aspects allow for viral entry]?
 
+Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052]. 
+
```

Comment:
```suggestion
```
You don't need spaces between each sentence, just between paragraphs (but you did a great job of separating the sentences onto separate lines, which really helps for editing!)
You can modify this by navigating to the "Files changed" tab at the top, then clicking the three dots below the green "Review changes" button. From there you will be able to "Edit file" and then commit directly to your branch (anskelly:patch-1). This will automatically apply your changes to the current PR!

Reply:
Thank you very much! Sorry for the hassle... I just went ahead and deleted the extra spaces (I think)!

Reply:
@rando2 @anskelly 
It looks great! I think my previous #199 can be ignored.

Reply:
@jinhui2 Thanks! I think you had some really great references! Like the JVI paper from Fang Li's group and the Wang et al. Cell paper! Maybe we can incorporate those?

Reply:
@anskelly Yes. I think it's a great idea to incorporate those papers. Thank you!

Reply:
@rando2 Thank you very much for your help! I think it would be great if we could cite the following 2 references after the first sentence in this PR: [@doi:10.1016/j.cell.2020.03.045] and [@doi:10.1128/jvi.00127-20]. Should I go ahead and add them on my end? Is there a way to make sure @jinhui2 gets credit for these?

Reply:
Hi @anskelly, thank you for the great question! If @jinhui2 sees this and is able to submit a [suggestion](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#the-how-to-of-review) GitHub will actually record them as the author! I can do something tricky on the command line to try to make it look like @jinhui2 submitted it, but this way is a lot easier!